lib/rubocop/cop/rails/active_record_aliases.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Finds usage of `update_attributes`/`update_attributes!`. Prefer
+      # the unaliased `update`/`update!`.
+      #
+      # @example
+      #   #bad
+      #   Book.update_attributes!(author: 'Alice')
+      #
+      #   #good
+      #   Book.update!(author: 'Alice')
+      class ActiveRecordAliases < Cop
+        ALIASES = {
+          update_attributes: :update,
+          update_attributes!: :update!
+        }.freeze
+
+        def on_send(node)
+          ALIASES.each do |bad, good|
+            next unless node.method?(bad)
+
+            add_offense(node,
+                        message: message(bad, good),
+                        location: :selector,
+                        severity: :warning)
+            break
+          end
+        end
+
+        def autocorrect(_node)
+          ->(corrector) { corrector.replace('update_attributes', 'update') }
+        end
+
+        private
+
+        def message(bad, good)
+          "Prefer #{good} over #{bad}."
+        end
```

Comment:
It would be better to follow the convention of using `MSG`.
https://github.com/bbatsov/rubocop/blob/v0.52.1/lib/rubocop/cop/rails/active_support_aliases.rb#L23